src/stream/src/executor/backfill/cdc/cdc_backfill.rs

```diff
@@ -272,6 +272,17 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
                                                 paused = false;
                                                 valve.resume();
                                             }
+                                            Mutation::Throttle(some) => {
```

Comment:
I remember there were discussions about using a separate channel for rate limit, instead of relying on barrier. Do we have any conclusion/follow-up about that?

Reply:
No updates currently. We don't see very strong motivation to have a separate control channel.

Reply:
I remembered the proposal was to use the same way as system parameters so that we can bypass barrier injection when barrier has already piled up.

Reply:
Separate control channel has the following benefits:
1. Alter rate limit can be done instantly by directly messaging the actors.
2. Decouple drop / cancel from barrier (requires further investigation).

Biggest benefit is from (2) IMO, if it's feasible. Since it is a big UX improvement. Requires an RFC to spec this out.

In terms of _functionality_ we can already support immediate execution of these commands under high barrier latency by triggering recovery. An improvement is #14700 in the meantime.